App/Redux/TextileEventsRedux.ts

```diff
@@ -1,42 +1,34 @@
 import { createAction, getType, ActionType } from 'typesafe-actions'
-import { NodeState } from '@textile/react-native-sdk'
 import { RootState } from './Types'
 
+export enum NodeState {
+  stopped = 'stopped',
+  started = 'started',
+  online = 'online'
+}
```

Comment:
No more `NodeState` model provided by the sdk, just event subscriptions for node started, stopped, etc, so we model the state here to store in redux and use in the app.